source/fuzz/protobufs/spvtoolsfuzz.proto

```diff
@@ -517,6 +520,28 @@ message TransformationAddDeadContinue {
 
 }
 
+message TransformationAddFloatCastSynonym {
+
+  // Inserts instructions to the module that convert
+  // |synonym_id| to float and back, and adds a fact that
+  // the converted id and |synonym_id| are synonymous.
+
+  // Id to apply conversion to. This must be a result id of vector or integer type.
+  // If the type is vector, its components must have an integer type.
+  uint32 synonym_id = 1;
```

Comment:
I don't think that synonym_id is a good name, since the id is not a synonym in advance of applying this transformation.  integer_id might be better.

Reply:
`synonym_id` might be a result id of instruction of vector or floating-point type. Thus, I think `integer_id` is not a good name either. I'll rename it to `result_id`.

Reply:
Fixed in fa3db6d.